### PR TITLE
#159965296 Configure App for Heroku

### PR DESCRIPTION
**Pull request by pngg**
#### What does this PR do?
Configures App for deployment on Heroku

#### Description of Task to be completed
- Setup app for Heroku deployment
- Setup ```staging``` and ```production``` environments on Heroku

#### How should this be manually tested?
- Navigate to the following URLs
[`staging environment`](https://ah-magnificent6-staging.herokuapp.com/)
[`production environment`](https://ah-magnificent6.herokuapp.com/)

#### What are the relevant pivotal tracker stories?
[Setup and deploy base application to Heroku](https://www.pivotaltracker.com/story/show/159965296)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -91,3 +91,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
```
Comment (meshack-mbuvi) [inline .gitignore]: These files are not needed. Please confirm.

File: .gitignore [hunk 2]
```diff
@@ -91,3 +91,6 @@ ENV/
...
+.DS_Store
+.vscode
```
Comment (meshack-mbuvi) [inline .gitignore]: Remove these files.

File: authors/settings.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 """
 
 import os
+import django_heroku
```
Comment (parseen254) [inline authors/settings.py]: Avoid use of preset libraries, for your own learning well being.

Comment (pngg) [reply authors/settings.py]: Removed django_heroku package and now reconfigured for manual deploy.

**Comment by meshack-mbuvi** [comment]
Resolve conflicts.

File: requirements.txt [hunk 4]
```diff
@@ -1,8 +1,19 @@
...
 django-extensions==2.1.2
 djangorestframework==3.8.2
+gunicorn==19.9.0
+isort==4.3.4
+lazy-object-proxy==1.3.1
+mccabe==0.6.1
+psycopg2==2.7.5
+psycopg2-binary==2.7.5
 PyJWT==1.4.2
+pylint==2.1.1
```
Comment (parseen254) [inline requirements.txt]: @pngg please remove non-essential requirements eg pylint.

Comment (mikenthiwa) [reply requirements.txt]: @parseendavid we have removed the unnecessary packages

**Pull request merged**